src/configuration/evars.md

```diff
@@ -1,326 +1,457 @@
-## Configuration Variables
+## Evaluable Variables
 
-Below is a list of the most frequently used configuration variables. You can get a complete list by issuing `e` command without arguments. For example, to see all variables defined in the "cfg" namespace, issue `e cfg.` (mind the ending dot). You can get help on any eval configuration variable by using `el cfg.`
+Rizin incorporates "evaluable variables", which are environment variables that influence the behavior of analysis, visuals, assembly, and more.
```

Comment:
I would avoid calling them "environment variables".  That's too easy to confuse with OS environment variables.

Reply:
i will probably revert them back to `configuration variables` .

So it will be:
```
Rizin incorporates "evaluable variables", which are configuration variables that influence the behavior of analysis, visuals, assembly, and more.
```